app/src/main/res/layout/activity_reservation_space_parking.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@drawable/wallpaper_main_background"
+    android:paddingLeft="@dimen/padding_parking_space_reservation"
+    android:paddingRight="@dimen/padding_parking_space_reservation">
+
+    <TextView
+        android:id="@+id/title_parking_space_reservation"
+        style="@style/TitleParkingSpaceReservationStyle"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:text="@string/text_parking_space_reservation_title" />
+
+    <Button
+        android:id="@+id/button_parking_space_reservation_picker_begin"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_below="@id/title_parking_space_reservation"
+        android:text="@string/button_parking_space_reservation_begin" />
+
+    <TextView
+        android:id="@+id/text_parking_space_reservation_picker_begin"
+        style="@style/TextParkingSpaceReservationStartStyle"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content" />
+
+    <Button
+        android:id="@+id/button_parking_space_reservation_picker_end"
+        style="@style/ButtonParkingSpaceReservationEndStyle"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="@string/button_parking_space_reservation_end" />
+
+    <TextView
+        android:id="@+id/text_parking_space_reservation_picker_end"
+        style="@style/TextParkingSpaceReservationEndStyle"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content" />
+
+    <EditText
+        android:id="@+id/input_parking_space_reservation_place"
+        style="@style/InputParkingSpaceReservationPlaceStyle"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:autofillHints=""
```

Comment:
for what are you using this?

Reply:
It was for a warning that says that I have to add this attribute even if my version is lower than 26. I searched for a solution and I added this in empty if I'm not going to use. I can delete this if you want!